packages/twenty-front/src/pages/settings/roles/components/RoleWorkspaceMemberPickerDropdown.tsx

```diff
@@ -0,0 +1,135 @@
+import { CoreObjectNameSingular } from '@/object-metadata/types/CoreObjectNameSingular';
+import { useFindManyRecords } from '@/object-record/hooks/useFindManyRecords';
+import { DropdownMenuItemsContainer } from '@/ui/layout/dropdown/components/DropdownMenuItemsContainer';
+import { DropdownMenuSearchInput } from '@/ui/layout/dropdown/components/DropdownMenuSearchInput';
+import styled from '@emotion/styled';
+import { t } from '@lingui/core/macro';
+import { ChangeEvent, useState } from 'react';
+import { Avatar } from 'twenty-ui';
+import { WorkspaceMember } from '~/generated-metadata/graphql';
+
+const StyledWorkspaceMemberSelectContainer = styled.div`
+  max-height: ${({ theme }) => theme.spacing(50)};
+  overflow-y: auto;
+`;
+
+const StyledEmptyState = styled.div`
+  align-items: center;
+  color: ${({ theme }) => theme.font.color.light};
+  display: flex;
+  font-size: ${({ theme }) => theme.font.size.md};
+  font-weight: ${({ theme }) => theme.font.weight.regular};
+  height: ${({ theme }) => theme.spacing(8)};
+  justify-content: flex-start;
+  padding: ${({ theme }) => theme.spacing(2)};
+`;
+
+const StyledWorkspaceMemberItem = styled.div`
+  align-items: center;
+  cursor: pointer;
+  display: flex;
+  font-size: ${({ theme }) => theme.font.size.md};
+  gap: ${({ theme }) => theme.spacing(2)};
+  min-width: ${({ theme }) => theme.spacing(45)};
+  padding: ${({ theme }) => theme.spacing(2)};
+
+  &:hover {
+    background: ${({ theme }) => theme.background.tertiary};
+  }
+`;
+
+const StyledWorkspaceMemberName = styled.div`
+  color: ${({ theme }) => theme.font.color.secondary};
+`;
+
+const StyledSearchInput = styled(DropdownMenuSearchInput)`
+  margin: 0;
```

Comment:
curious why we have this design change here, let's build it like the other dropdowns, designs should be consistent